components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -215,4 +220,26 @@ public String getFriendlyName() {
     public String getName() {
         return GoogleOAuth2AuthenticationConstant.GOOGLE_CONNECTOR_NAME;
     }
+
+    @Override
+    public String getClaimDialectURI() {
+        String claimDialectUri = GoogleOAuth2AuthenticationConstant.OIDC_CLAIM_DIALECT_URI;
+        AuthenticatorConfig authConfig = FileBasedConfigurationBuilder.getInstance().getAuthenticatorBean(getName());
+        if (authConfig != null) {
+           Map<String, String> parameters = authConfig.getParameterMap();
+           if (parameters != null && parameters.containsKey(GoogleOAuth2AuthenticationConstant.
+                   PARAMETER_VALUE_FOR_CLAIM_DIALECT)) {
+               claimDialectUri = parameters.get(GoogleOAuth2AuthenticationConstant.PARAMETER_VALUE_FOR_CLAIM_DIALECT);
+           } else {
+               if (log.isDebugEnabled()) {
+                   log.debug("Found no Parameter map for connector " + getName());
+               }
+           }
+        } else {
+            if (log.isDebugEnabled()) {
+                log.debug("Error Reading Parameters for connector " + getName());
```

Comment:
Add more information to this log. Just saying 'error reading param' will not help in debugging. Mention something like FileBasedConfigBuilder returned null AuthenticatorConfigs for the connector.

Reply:
Fixed it